app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -26,7 +26,7 @@ define(function (require, exports, module) {
 
       // When account or user fields are needed,
       // they can be added as part of the resumeTokenInfo
-      var resumeTokenInfo = _.extend(
+      var resumeTokenInfo = Cocktail.mixin(
```

Comment:
Hmm, Cocktail is meant to be used with Views, I'm not sure of the repercussions of doing this. Can you change this back to `_.extend`?